File: app/src/main/java/com/fractalwrench/crazycats/common/TextUtils.java
package com.fractalwrench.crazycats.common;

public final class TextUtils {

    private TextUtils() {
        throw new AssertionError();
    }

    /**
     * Returns true if the {@link CharSequence} is empty or null
     *
     * @param text the input text
     * @return true if empty
     */
    public static boolean isEmpty(CharSequence text) {
        return text == null || text.length() == 0;
    }

}

File: app/src/main/java/com/fractalwrench/crazycats/common/TextUtils.kt
package com.fractalwrench.crazycats.common

class TextUtils private constructor() {

    init {
        throw AssertionError()
    }

    companion object {

        /**
         * Returns true if the [CharSequence] is empty or null

         * @param text the input text
         * *
         * @return true if empty
         */
        fun isEmpty(text: CharSequence?): Boolean {
            return text == null || text.isEmpty()
        }
    }

}
